pkg/config/testing.go

```diff
@@ -16,17 +16,16 @@ import (
 var mainConfig = `
 {
     "contractAddress": "0x0Ba45A8b5d5575935B8158a88C631E9F9C95a2e5",
-    "nodeURL": "https://mainnet.infura.io/v3/7f11ed6dfxxxxxxxxxxxxxxxxx",
     "databaseURL": "http://localhost7545",
-    "publicAddress": "3808C441402D0418C52237B697bBC5D18c84839A",
-    "privateKey": "A000000000000000000000000000000000000000000000000000000000000000",
+    "publicAddress": "92f91500e105e3051f3cf94616831b58f6bce1e8",
     "serverHost": "localhost",
     "serverPort": 5000,
     "trackerCycle": 1,
     "trackers": {},
     "dbFile": "/tellorDB",
     "requestTips": 1,
-    "configFolder": "` + filepath.Join("..", "..", "configs") + `"
+    "configFolder": "` + filepath.Join("..", "..", "configs") + `",
+    "envFile": "` + filepath.Join("..", "..", "configs/.env.example") + `"
```

Comment:
```suggestion
    "envFile": "` + filepath.Join("..", "..", "configs",".env.example") + `"
```

The idea with `filepath.Join` is that under Linux the delimiter is `/` and under windows it uses `\`